### PR TITLE
releng: Image promotion for Kubernetes v1.19.3

### PR DESCRIPTION
**Pull request by mkorbi**
Logs from the cloudbuild:

- [Mock Stage](https://console.cloud.google.com/cloud-build/builds/01b2a651-bd67-4ce9-bd7b-82c44391e7fa)
- [Mock Release](https://console.cloud.google.com/cloud-build/builds/2678a968-b225-4069-bfe3-9ff23f183657)
- [Stage](https://console.cloud.google.com/cloud-build/builds/bcf98f29-68ad-4b5a-8746-4a51eca5bf2a)

@kubernetes/release-engineering
/assign @justaugustus @saschagrunert
/hold

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml [hunk 1]
```diff
@@ -976,6 +1006,96 @@
     "sha256:e15927e26974af0e066018b056c5f51e514347950e164e4e5be822ed723ee7e0": ["v1.17.10"]
     "sha256:f5de4ad913ffd66ceee261f40fecaec852342168596c9e6d2d8c6fe9f1cd7359": ["v1.18.9"]
     "sha256:f86066539c4a8fb82715e049ef13c0161dbc3b534881b2315476b254b0556733": ["v1.19.1"]
+- name: mock/conformance
+  dmap:
+    "sha256:c9b97cfe2af7e54e8203fdc3fbc6e5151d32066cab5fd1e3c0a929c32b83af65": ["v1.19.3"]
```
Comment (saschagrunert) [inline k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:1011]: Okay this does not look correct. @puerco do you have an idea why those are listed here?

**Review by saschagrunert — CHANGES_REQUESTED** [review]

**Comment by saschagrunert** [comment]
The promoted images of `v1.19.4-rc.0` are missing, too.

**Review by cpanato — CHANGES_REQUESTED** [review]
need to add the next which will be `"v1.19.4-rc.0"`

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml (hunk 1 above)
Comment (hasheddan) [reply k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:1011]: @saschagrunert I am guessing this was done without using `krel` as these previously had to be manually removed :+1:

Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:1011]: need to remove the mock one as well

**Review by saschagrunert — APPROVED** [review]
/lgtm
/approve

**Pull request merged**